packages/pn-pa-webapp/src/pages/NotificationDetail.page.tsx

```diff
@@ -88,7 +96,7 @@ const NotificationDetail = () => {
   const recipientsWithAltNoticeCode = recipients.filter(
     (recipient) => recipient.payment?.noticeCodeAlternative
   );
-  const { t } = useTranslation(['common', 'notifiche']);
+  const { t } = useTranslation(['common', 'notifiche', 'appStatus']);
```

Comment:
The appStatus namespace is not used, can be removed?

Reply:
in fact the 'appStatus' is needed since it is used in components in `pn-commons` that are used by `NotificationDetail.page`. I will add a comment about this fact.